solvebio/query.py

```diff
@@ -933,6 +936,9 @@ def join(self, query_b, key, key_b=None, prefix="b_", always_prefix=False):
         # Add to any existing target fields
         new_query._target_fields += target_fields
 
+        # Extend explode_fields with prefixed fields to be exploded
+        explode_fields = [field['name'] for field in new_query._target_fields if 'join' not in field['name']]
```

Comment:
without that, the result will contain lists as values for all previous joins before the last one. 

```
{'c_id': '1', 'name': 'Francis Crick', 'b_id': ['1'], 'b__commit': ['1329009293354399140'], 'c__commit': '1329009293354399140', 'b_name': ['Francis Crick'], '_commit': '1329011331274567012', 'id': '1', 'c_name': 'Francis Crick'}
{'c_id': '2', 'name': 'James Watson', 'b_id': ['2', '2'], 'b__commit': ['1329009293354399140', '1329009293354399140'], 'c__commit': '1329009293354399140', 'b_name': ['James Watson', 'James Watson'], '_commit': '1329011331274567012', 'id': '2', 'c_name': 'James Watson'}
{'c_id': '2', 'name': 'James Watson', 'b_id': ['2', '2'], 'b__commit': ['1329009293354399140', '1329009293354399140'], 'c__commit': '1329009293354399140', 'b_name': ['James Watson', 'James Watson'], '_commit': '1329011331274567012', 'id': '2', 'c_name': 'James Watson'}
```

`explode` function requires fields that will be parsed in order to make tuples with their values. 

https://github.com/solvebio/api.solvebio.com/blob/c375407489cd4c0da7c4fc561ed016138fa6036a/api/annotations/functions/records.py#L21

Reply:
Ah ok, need to think about this some more then! Maybe we should store the list of explode fields in the query and pass that through to the chained queries? would that work? i.e. if `self._join_fields` then add those to the explode fields list. I don't like this check for 'join' in the field name because it will break if the field has join in it.

Reply:
Seems it works. I added them to `self._explode_fields` attribute that is going to be extended by all fields that are created within each additional join function.